addons/FlashCards/src/presenter.js

```diff
@@ -620,6 +633,77 @@ function AddonFlashCards_create(){
         
     };
 
+    presenter.sendAudioEvent = function(eventName) {
+        var eventData = {
+            'source': presenter.configuration.addonID,
+            'item': '',
+            'value': eventName,

```

Comment:
Czy jesteś pewny, że dla każdego eventu pod kluczem "value" ma być przekazywana ta informacja ?
Addony takie jak Audio (o którym mowa w tickecie) lub TextAudio na zakończenie nagrywania wysyłają event, gdzie "end" jest wysyłane pod kluczem "item". Natomiast Audio dodane jako insert audio w module text wysyła "end" pod kluczem "value" tak jak Ty masz. Mam wrażenie, że to ważne, gdyż użyto sformułowania "Potrzebne do wzajemnego wyłączania audio..." w opisie zadania.
Sprawdź moją lekcje: https://test-8817-dot-mauthor-dev.ew.r.appspot.com/present/4660735810469888#2